packages/lms/src/lib/components/navigation/CourseNav.svelte

```diff
@@ -31,22 +36,27 @@
 		font-weight: 600rem;
```

Comment:
Didn't this get fixed somewhere or am I drunk?

Reply:
Yeah got fixed on content-dependencies, this probably just need the master merge into it to update